spec/dummy/db/structure.sql

```diff
@@ -172,11 +167,18 @@ CREATE UNIQUE INDEX index_my_records_on_wisdom ON my_records USING btree (wisdom
 
 
 --
--- PostgreSQL database dump complete
+-- Name: index_vehicles_on_make_and_name; Type: INDEX; Schema: public; Owner: -
 --
 
-SET search_path TO "$user", public;
+CREATE UNIQUE INDEX index_vehicles_on_make_and_name ON vehicles USING btree (make, name);
 
-INSERT INTO schema_migrations (version) VALUES ('20160419103547'), ('20160419124138');
 
+--
+-- PostgreSQL database dump complete
+--
+
+SET search_path TO "$user", public;
 
+INSERT INTO "schema_migrations" (version) VALUES
+('20160419103547'),
+('20160419124138');
```

Comment:
👍 
I removed the entire file. 
The specs are green without it.
In any case, running the mentioned actions in the README will regenerate this file if necessary.